source/io/TlsOptions.cpp

```diff
@@ -27,25 +27,29 @@ namespace Aws
             TlsContextOptions TlsContextOptions::InitDefaultClient() noexcept
             {
                 TlsContextOptions ctxOptions;
-                aws_tls_ctx_options_init_default_client(&ctxOptions.m_options);
+                aws_tls_ctx_options_init_default_client(&ctxOptions.m_options, aws_default_allocator());
```

Comment:
This needs to honor the configured allocator.

Reply:
Fixed.